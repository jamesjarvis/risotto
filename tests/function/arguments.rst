func testFunction1() int {
    return 0
}

testFunction1()

func testFunction2(a1 int, a2 int, a3 int) string {
    println(a1)
    println(a2)
    println(a3)

    return a1 + " " + a2 + " " + a3
}

println(testFunction2(1, 2, 3))

func testFunction3() {
    println("test")
}

testFunction3()

println("done")
