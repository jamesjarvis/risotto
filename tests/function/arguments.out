1
2
3
1 2 3
test
done
